plugins/SitesManager/lang/en.json

```diff
@@ -34,6 +34,10 @@
         "GlobalListExcludedUserAgents_Desc": "If the visitor's user agent string contains any of the strings you specify, the visitor will be excluded from Matomo.",
         "GlobalSettings": "Global settings",
         "GlobalWebsitesSettings": "Global websites settings",
+        "InstallationGuides": "Installation Guides",
+        "InstallationGuidesIntro": "We provide step by step guides explaining how to embed the tracking code for some popular CMS and website builders:",
+        "ExtraInformationNeeded": "Information you might need for some integrations:",
```

Comment:
could write it more clearly like "To setup Matomo on your system, you may need the following information"